src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -437,4 +437,7 @@
   <data name="ERR_NameNotInContext" xml:space="preserve">
     <value>The name '{0}' does not exist in the current context.</value>
   </data>
+  <data name="HideBase" xml:space="preserve">
+    <value>Hide base member</value>
```

Comment:
Spec section 10.3.4 calls it hiding, so let's go with that.

---
A class-member-declaration is permitted to declare a member with the same name or signature as an inherited member. When this occurs, the derived class member is *said to hide* the base class member. Hiding an inherited member is not considered an error, but it does cause the compiler to issue a warning. To suppress the warning, the declaration of the derived class member can include a new modifier to indicate that the derived member is intended to hide the base member. This topic is discussed further in §3.7.1.2.
If a new modifier is included in a declaration that doesn’t hide an inherited member, a warning to that effect is issued. This warning is suppressed by removing the new modifier.